roles/keycloak_quarkus/tasks/fastpackages.yml

```diff
@@ -1,17 +1,7 @@
 ---
```

Comment:
please include fastpackages.yml (former version calling rpm) when on RedHat family, and do the direct builtin package otherwise